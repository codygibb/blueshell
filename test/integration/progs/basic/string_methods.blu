s := ''
print s.len()
s = "hello world"
print s.len()

print s[3]
print s.substr(3)
print s.substr(3, 7)

