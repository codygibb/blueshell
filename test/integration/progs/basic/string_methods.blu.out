0
11
l
lo world
lo w
